activemodel/lib/active_model/type/boolean.rb

```diff
@@ -1,9 +1,22 @@
 module ActiveModel
   module Type
-    class Boolean < Value # :nodoc:
+    # == Active \Model \Type \Boolean
+    #
+    # A class representing +boolean+ with functionality for coercion to and from
+    # a database or user input.
+    #
+    # === Coercion
+    # Values set from user or database input will first be coerced into a
```

Comment:
From a public API point of view, how database input is coerced is entirely irrelevant. We only need to focus on what happens during assignment from rubyland